src/__tests__/wait-for-dom-change.js

```diff
@@ -0,0 +1,172 @@
+import {waitForDomChange} from '../'
+// adds special assertions like toBeTruthy
+import 'jest-dom/extend-expect'
+import {render} from './helpers/test-utils'
+
```

Comment:
The code below could be simplified:

```javascript
const skipSomeTime = (delayMs) => new Promise(resolve => setTimeout(resolve, delayMs))

// Using `setTimeout` >30ms instead of `wait` here
// because `mutationobserver-shim` uses `setTimeout` ~30ms.
const skipSomeTimeForMutationObserver = (delayMs = 50) => skipSomeTime(delayMs)
```